com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/HDShaderGUI.cs

```diff
@@ -17,14 +17,25 @@ namespace UnityEditor.Rendering.HighDefinition
     // of the material you must use Material UI Blocks, examples of doing so can be found in the classes UnlitGUI,
     // LitGUI or LayeredLitGUI.
 
-    abstract class HDShaderGUI : ShaderGUI
+    /// <summary>
+    /// Use this class to build your custom Shader GUI for HDRP.
+    /// You'll then be able to use the class that inherit from HDShaderGUI in the ShaderGraph Custom EditorGUI field.
```

Comment:
You can use a class that inherits from HDShaderGUI in the Shader Graph **Custom EditorGUI** field.